### PR TITLE
responsive-web-design-1

### PR DESCRIPTION
**Pull request by Jon-Bernal**
@YanrongXu Here's the first pull

### CODE REVIEW

**Comment by Jon-Bernal** [comment]
My browser window won't shrink down to below 560 pixels. So I went with the break point at 600px instead of 500px so I could see what I was changing.  It threw some things off, but I did my best with what I had.

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+
```
Comment (YanrongXu) [inline contact.html:2]: the ways your write code, is the best i have seen

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="contact.html">Contact</a>
+    </div>
+    <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+  </nav>
+
+  <header>
+    <img src="img/services-header.jpg" alt="">
```
Comment (YanrongXu) [inline contact.html:32]: If possible please have alt, so the website is more accessible to blind

File: css/contact.css [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+h3, p {
+    font-size: .8rem;
+    line-height: 1rem
+}
+
+/* ******* Nav ******* */
+nav {
+	width: 100%;
```
Comment (YanrongXu) [inline css/contact.css:99]: great on use both id and  class

File: css/index.css [hunk 4]
```diff
@@ -75,120 +75,128 @@ h1, h2, h3, h4, h5 {
...
 	flex-direction: row;
 	justify-content: space-around;
 	align-items: center;
-	margin-top: 20px;
-	padding: 10px 0;
 }
 
-nav a img {
-    justify-self: flex-end;
+@media (max-width: 800px){
```
Comment (YanrongXu) [inline css/index.css:103]: smart way by use flex-direction for tablet

File: css/index.css [hunk 5]
```diff
@@ -75,120 +75,128 @@ h1, h2, h3, h4, h5 {
...
 
 nav a {
+	text-decoration: none;
 	color: gray;
-    font-size: 1rem;
-    /* font-weight: bold; */
 }
 
+@media (max-width: 600px) {
+	.nav-left {
```
Comment (YanrongXu) [inline css/index.css:141]: same, your are smart on how css work

File: css/index.css [hunk 6]
```diff
@@ -166,36 +199,73 @@ header div a {
...
 
+@media (max-width: 800px){
+	.section2 {
+		justify-content: space-evenly;
+		margin-left: 3%;
+		margin-right: 3%;
+	}
+}
+
+.section2 div p {
```
Comment (YanrongXu) [inline css/index.css:310]: if possible, please put some tab on those line, so it can look pretty

File: css/services.css [hunk 7]
```diff
@@ -271,11 +271,26 @@ main {
     /* display: flex;
     flex-direction: column; */
 }
+@media (max-width: 600px){
 
+    .about div  {
+        margin-top: 60px;
+        padding: 0 5%;
+        padding-bottom: 30px;
```
Comment (YanrongXu) [inline css/services.css:279]: the web look great on every form

**Pull request merged**